src/layouts/default.vue

```diff
@@ -1,14 +1,15 @@
 <template>
-  <div class="app relative grid">
+  <div class="app flex min-h-screen flex-col">
```

Comment:
Would it be better to not touch the `default` wrapper styles until we finish header/footer implementations and enable them? I'm afraid this might break the iframe version inside wordpress.org (I haven't checked, though).

Reply:
I think this breaks the sidebar layout for the old header version of the site. If you turn off the `new_header` flag, the sidebar hides a part of the page:
<img width="949" alt="Screen Shot 2022-09-19 at 9 11 56 PM" src="https://user-images.githubusercontent.com/15233243/191085524-52da4381-68c8-4ba7-86f0-aed17bfe95ba.png">

Reply:
Oh, I see, it's on a very specific breakpoint. Nice find @obulat !

Reply:
That's how it behaves in prod. I haven't changed it.

<img width="965" alt="Screenshot 2022-09-20 at 7 08 09 AM" src="https://user-images.githubusercontent.com/16580576/191158666-726b0e79-b369-4d99-98c5-7c62d77b568d.png">

Reply:
@dhruvkb and @obulat I'm logging off for the night but it looks like this regression at this breakpoint was introduced in https://github.com/WordPress/openverse-frontend/pull/1804. Could we get a critical issue and pr fix created? It must just be an issue of using the wrong media query in one place, likely a small problem.

Reply:
@obulat this is the same behaviour as prod and staging.

<img width="965" alt="Screenshot 2022-09-20 at 7 27 20 AM" src="https://user-images.githubusercontent.com/16580576/191160892-23c2ac5d-a255-4722-92b4-7d8a3fb7a46f.png">
<img width="965" alt="Screenshot 2022-09-20 at 7 27 46 AM" src="https://user-images.githubusercontent.com/16580576/191160940-0bdaf3bc-c5ec-4622-bdb3-7de45a7f8349.png">